phalcon/mvc/collection.zep

```diff
@@ -675,6 +675,37 @@ abstract class Collection implements EntityInterface, CollectionInterface, Injec
 		return !count(messages);
 	}
 
+	/**
+	 * Check whether validation process has generated any messages
+	 *
+	 *<code>
+	 * use Phalcon\Mvc\Model\Validator\ExclusionIn as ExclusionIn;
```

Comment:
From Phalcon 4.0.x this namespace like `Phalcon\Mvc\Model\Validator`.

You should use `Phalcon\Validation\*` namespace. And then this looks like this:

```php
 public function validation()
 {
    $validation = new \Phalcon\Validation();
    $validation->add(
    'status',
	 new ExclusionIn(
           [
                "domain" => ["A", "I"],
           ]
    )

    return $this->validate($validation);
);
```

So i doubt in necessity of this method, or at least this comment.